README.md

```diff
@@ -1,2 +1,70 @@
 # Telnyx Video SDK for iOS
 This repository contains releases of Telnyx's Video SDK for iOS. These releases can be installed as a CocoaPod or manually into your project.
+
+## Adding Telnyx Video SDK to your iOS Client Application:
```

Comment:
@guillermobattistel I'm thinking we should keep the instructions here in this repo as simple as possible and scoped to installing the release. 

The instructions you have here are really helpful though. Can we move this section over to telnyx-meet-ios and add a link to that here?

So...
1. telnyx-meet-ios README have a section for `Adding the Telnyx Video SDK to your app`
2. Inside this README include the instructions to install the Cocoa like you on `line 7`

Reply:
I think that splitting install docs will produce confusion. Let's discuss this.

Reply:
SGTM! 🙏